src/plugins/flexibleLayout/components/frame.vue

```diff
@@ -89,6 +88,18 @@ export default {
             this.unsubscribeSelection = this.openmct.selection.selectable(this.$refs.frame, context, false);
         },
         initDrag(event) {
+            let type = this.openmct.types.get(this.domainObject.type),
```

Comment:
My thoughts were it would be better to have less nodes in the DOM. But if you prefer your approach, I can change. In the future I was hoping to create a ghost image API which could be used by other views as well.

Reply:
Fair enough. Fewer nodes in the DOM is definitely not a bad thing.